wrappers/IWearLogger/src/IWearLogger.cpp

```diff
@@ -25,8 +27,20 @@ constexpr double DefaultPeriod = 0.01;
 namespace wearable {
     namespace wrappers {
         struct IWearLoggerSettings;
+
+        enum class LoggerLevel
+        {
+            MATLAB = 0,
+            YARP,
+            MATLAB_YARP,
+            NONE,
```

Comment:
This is a bit weird, especially the double option. It does not scale gracefully with another logging method :/ Though, as already commented by @lrapetti, the best thing would maybe creating a base device with a shared configure method for the common options and then specialize it for the different NWSs (network server, i.e. the preferred terminology for a wrapper).

Reply:
> @lrapetti, the best thing would maybe creating a base device with a shared configure method for the common options and then specialize it for the different NWSs (network server, i.e. the preferred terminology for a wrapper).

yess, anyway, as I said before I don't think at this stage there is plan to address it, so I think we can proceed with this device implementation for the time-being.

Reply:
Yep, it takes a considerable amount of work, not worth at this stage.

Reply:
As discussed with @lrapetti and @prashanthr05 this is a lean way to get the wearable data streamed using IWear logger device. Definitely this pipeline can be improved better with further refactoring, but this needs a bit more of planning and time to do it.